apps/consulting/posts/openai-pseudocode.md

```diff
@@ -0,0 +1,225 @@
+---
+title: 'Collaborative code generation with LLMs'
+published: March 27, 2023
+author: chris-ostrouchov
+description: 'As an experiment we developed a python package pseudocode which much like its name implies allows developers to annotate their code with pseudocode and produce testable code generated by LLMs without the user writing any code.'
+category: ["ML", "LLM", "python", "openai"]
```

Comment:
Likely will want to include at least one of the [already-existing categories][categories.json] so that the "see these other related articles" component at the bottom of the blog page will show some posts. `Artificial Intelligence` seems the most relevant to me.

Any categories you wish to add here that don't exist yet in [`categories.json`][categories.json] will need to be added there, following the object format defined there.


[categories.json]: https://github.com/Quansight/Quansight-website/blob/develop/apps/consulting/posts/categories.json

Reply:
@costrouc add AI as suggested and modify `categories.json`

Reply:
Found out there is already an `artificial-intelligence` category. Used that.